src/index.js

```diff
@@ -0,0 +1,146 @@
+    setTasksLocalStorage(tasksContainer);
+  });
+
+  const menuIcon = new Image();
+  menuIcon.src = moreIcon;
+  menuIcon.setAttribute('alt', 'menu-icon');
+  menuIcon.classList.add('more-icon');
+  taskContainer.appendChild(menuIcon);
+
+  const delIcon = new Image();
+  delIcon.src = deleteIcon;
+  delIcon.setAttribute('id', 'deleteTask');
+  delIcon.classList.add('check-box', 'reload-icon', 'delete-icon', 'hide');
+  delIcon.setAttribute('alt', 'delete-icon');
+  taskContainer.appendChild(delIcon);
+
+  descriptionContainer.addEventListener('focus', () => {
+    taskContainer.classList.toggle('focus-task');
+    menuIcon.classList.toggle('hide');
+    delIcon.classList.toggle('hide');
+  });
+
+  descriptionContainer.addEventListener('blur', () => {
+    taskContainer.classList.toggle('focus-task');
+    menuIcon.classList.toggle('hide');
+    delIcon.classList.toggle('hide');
+  });
+
+  delIcon.addEventListener('click', () => {
+    tasksContainer = deleteSelectedTask(delIcon.parentElement, tasksContainer);
+    setTasksLocalStorage(tasksContainer);
+  });
+
+  tasksInnerContainer.appendChild(taskContainer);
+};
+
+const displayTasks = (array) => {
+  for (let i = 0; i < array.length; i += 1) {
+    createTaskHTML(array[i].description, array[i].completed);
+  }
+};
+
+const upwardOrderArray = (array) => {
+  for (let i = 0; i < array.length - 1; i += 1) {
+    for (let j = i + 1; j < array.length; j += 1) {
+      if (array[i].index > array[j].index) {
+        [array[j], array[i]] = [array[i], array[j]];
+      }
+    }
+  }
+  return array;
+};
+
+const setReloadIcon = () => {
+  const formContainer = document.querySelector('.title-container');
+  const reIcon = new Image();
+  reIcon.src = reloadIcon;
+  reIcon.setAttribute('id', 'reloadButton');
+  reIcon.classList.add('check-box', 'reload-icon');
+  reIcon.setAttribute('alt', 'reload-icon');
+  formContainer.appendChild(reIcon);
+};
+
+const setEnterIcon = () => {
+  const formContainer = document.querySelector('.form-container');
+  const enIcon = new Image();
+  enIcon.src = enterIcon;
+  enIcon.setAttribute('id', 'enterForm');
+  enIcon.classList.add('check-box', 'reload-icon');
+  enIcon.setAttribute('alt', 'enter-icon');
+  formContainer.appendChild(enIcon);
+};
+
+setEnterIcon();
+setReloadIcon();
+
+window.onload = () => {
+  if (tasksContainer != null && tasksContainer.length > 0) {
+    tasksContainer = upwardOrderArray(tasksContainer);
+    displayTasks(tasksContainer);
+  } else {
+    tasksContainer = [];
+  }
+};
+
+const enterButton = document.getElementById('enterForm');
+const clearButton = document.getElementById('clearButton');
+
+enterButton.addEventListener('click', () => {
+  const newDescription = document.getElementById('taskForm').value;
+  const newTask = addTask(newDescription, tasksContainer);
+  createTaskHTML(newTask.description);
+  document.getElementById('taskForm').value = '';
+});
```

Comment:
Following the previous comment: [change div to form](https://github.com/Yothu/todo-list-feedback/pull/1/files#r786307973)
This 'click' event listener can be used to submit the form too.
The script contained in this 'click' event listener would now be executed after listening to a 'submit' from the form.

Reply:
I created a new event listener for the `input text` where it reacts to a keypress while the `input text` is focused, that way it has the same effect as your approach, thanks for the review!.

Reply:
Very creative solution. Kudos!
Loved the console.log